tests/rules/property.js

```diff
@@ -252,6 +252,15 @@ eslintTester.run("property", rule, {
             code: "let msg = '<b>safe</b>'; let altMsg = 'also cool';  if (monday) { msg = altMsg; }; y.innerHTML = msg;",
             parserOptions: { ecmaVersion: 6 },
         },
+        {
+            code: "x.setHTML(evil)"
+        },
+        {
+            code: "x.setHTML(evil, { sanitizer: s })"
+        },
+        {
+            code: "x.setHTML(evil, { sanitizer: new Sanitizer()})"
+        }
```

Comment:
@mozfreddyb shouldn't this additional test cases be technically added to `tests/rules/method.js` instead of `tests/rules/property.js`?

I don't expect it to make a difference in practice (I expect the test case to still pass just fine when moved into the other test file), but `tests/rules/method.js` seems a better fit and consistent with the kind of test cases we added in both of them over the time.

wdyt?